common/djangoapps/enrollment/views.py

```diff
@@ -264,9 +267,12 @@ class EnrollmentListView(APIView, ApiKeyPermissionMixIn):
 
                If honor mode is not supported for the course, the request fails and returns the available modes.
 
-               A server-to-server call can be used by this command to enroll a user in other modes, such as "verified"
-               or "professional". If the mode is not supported for the course, the request will fail and return the
-               available modes.
+               A server-to-server call can be used by this command to enroll a user in other modes, such as "verified",
+               "professional" or "credit". If the mode is not supported for the course, the request will fail and
+               return the available modes.
+
+               The enrollment attributes are also set against the course mode if provided 'enrollment_attributes' are
```

Comment:
I'm not sure what "set against course mode" means.  not sure how this info differs from preceding sentence.  Can you clarify?